### ISSUE TITLE
Color contrast issue in top banner links in dark mode

### ISSUE THREAD

**Issue by joedolson** [wpa11yday/wpaccessibilityday#103]
Reported by GersonL: 

When using the dark mode on https://2023.wpaccessibility.day/, the link in the notice bar is white against the light sky blue background. The link text should be darker. Thanks.
